.github/workflows/tests.yml

```diff
@@ -45,7 +45,7 @@ jobs:
             - name: Install Go
               uses: actions/setup-go@v4
               with:
-                  go-version: 1.19
+                  go-version: 1.20
```

Comment:
This is a yaml literal float equal to `1.2`---probably need to quote this.

Reply:
Oh fun.